simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/station/handlers/ocpp/GetVariablesRequestHandlerTest.java

```diff
@@ -65,13 +68,33 @@ void verifyCallResult() throws JsonProcessingException {
                 .withAttributeValue(String.valueOf(DEFAULT_HEARTBEAT_INTERVAL))
                 .build();
 
-        String expectedCallResultJson = createCallResult()
-                .withMessageId(DEFAULT_MESSAGE_ID)
-                .withPayload(getVariablesResponse)
-                .toJson();
+        GetVariableResult expectedResult = getVariablesResponse.getGetVariableResult().get(0);
 
-        assertThat(messageCaptor.getValue().getData().get()).isEqualTo(expectedCallResultJson);
+        GetVariableResult actualResult = messageCaptor.getValue().getGetVariableResult().get(0);
 
+        assertAll(
+                () -> assertThat(actualResult.getAttributeStatus()).isEqualTo(expectedResult.getAttributeStatus()),
+                () -> assertThat(actualResult.getComponent().getName()).isEqualTo(expectedResult.getComponent().getName()),
```

Comment:
I don't have the simulator code checked out, but can these assertions be replaced with a single equals? i.e. expectedResult.isEqualTo(actualResult)?

Reply:
equals is not overwritten -- identity equality